conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -5300,21 +5304,21 @@ msgid "Please contact your course team if you have any questions."
 msgstr ""
 
 #: lms/djangoapps/courseware/views/views.py
-msgid "Your certificate is available"
+msgid "We're working on it..."
 msgstr ""
 
 #: lms/djangoapps/courseware/views/views.py
-msgid "You've earned a certificate for this course."
+msgid ""
+"We're creating your certificate. You can keep working in your courses and a "
+"link to it will appear here and on your Dashboard when it is ready."
```

Comment:
Did you mean to add this? If so, add that they can also see it on their profile.

Reply:
Good catch. I did not!